### ISSUE TITLE
[Fluxstore] - How to set predefined Sate/Province in app address area? 

### ISSUE THREAD

**Issue by themeix** [inspireui/support#2217]
In the shipping_address.dart file, there have State/province input box but we need a Dropdown option otherwise its making problem in final checkout webview page if the text missmatch.

So how do i set the state list manullay there?
Your code,
```
             TextFormField(
                      controller: _stateController,
                      validator: (val) {
                        return val.isEmpty
                            ? S.of(context).streetIsRequired
                            : null;
                      },
                      decoration: InputDecoration(
                          labelText: S.of(context).stateProvince),
                      onSaved: (String value) {
                        address.state = value;
                      },
                    ),
```
Can you give me code where shows the state list automatically based on country or i can define there as string array.

2. Screenshots issues:
https://i.postimg.cc/xdd5QCCP/screenshot-622.png
3. Proof of my purchased:
https://raw.githubusercontent.com/themeix/temp/master/screenshot-589.png
3. Config files (optional)
Not required

**Comment by minhcasi** [comment]
Could you recommend which the endpoint API that supports to show State loading by country?

Also, move to the backlog for the next integration - thank you!

**Comment by themeix** [comment]
@minhcasi  Ref : https://stackoverflow.com/questions/33934028/using-the-woocommerce-api-to-get-the-country-state-lists/34000585

**Comment by minhcasi** [comment]
@themeix - thank you - it seems we need to build new Endpoint API to support for this feature.

@minhtruong315 - pls note for this improvement

**Comment by themeix** [comment]
Thanks to mentioning others issue here. So here is summery.
1) Profile update not working. Video Example : https://screencast-o-matic.com/watch/cYeecnxenq
2) Order note not working https://screencast-o-matic.com/watch/cYeeXoxfn9
3) All products not loading in Category page https://screencast-o-matic.com/watch/cYeeXYxfeq

**Comment by themeix** [comment]
I fixed the Order Note issue. 
In your app, order.dart,
there have the following code,

```
    if (kAdvanceConfig['EnableReview'] &&
        cartModel.notes != null &&
        cartModel.notes.isNotEmpty) {
      params["customer_note"] = cartModel.notes;
    }
```
I am not sure why you added EnableReview is there coz that is not related with that logic. I just removed that and its working fine. Actually i disabled review so note also stopped working..

**Comment by themeix** [comment]
Can you please help to fix the Profile update issue?

**Comment by minhcasi** [comment]
@themeix - confirm it is fixed on the latest code - we will update to our Gitlab so that you can download it tomorrow - thank you!

**Comment by themeix** [comment]
Just for your kind information, I fixed the category issue also by changing the endpoint url. I had to remove **SALE**, **FEATURE** parameter from the url.

**Comment by themeix** [comment]
And i forgot to inform, Points page showing Blank page.. Its not working too so if you will push update tomorrow please double-check that feature

**Comment by themeix** [comment]
>  you can download it tomorrow - thank you!

Waiting for the new version

**Comment by themeix** [comment]
Any update please?

**Comment by themeix** [comment]
No one replying here..  Anyone working on that issue please?????

**Comment by minhcasi** [comment]
@themeix - we are working hard to update thew new release on this week - thanks

**Comment by themeix** [comment]
@minhtruong315, Don't forget about #2261 that issue please

**Comment by minhtruong315** [comment]
I updated dropdown for state, it'll be available in next release. thanks

**Comment by themeix** [comment]
I can see you closed the issue but still waiting for the next release. You guys should help to fix the issue fast. I have been waiting since 17 days and doing struggle to get the bug fixed.

**Comment by minhcasi** [comment]
@themeix - new release will be available today - we will explain detail from the changelog - thanks

**Comment by themeix** [comment]
Got the new version today in codecanyon.. waiting for the docs

**Comment by themeix** [comment]
Installed new version today and i can see 
https://i.postimg.cc/0NH3ND75/screenshot-640.png

That is not working.. I did not change anything.. everything on download package and its not working

**Comment by themeix** [comment]
Profile update issue, Quantity issue not fixed in 1.6.6
https://screencast-o-matic.com/watch/cYeYoLyhxK
Please open that issue again to discuss and resolve that.. I dont want to create new ticket. Trying to get help you from last few weeks but still not resolved my issue.. Please give me High priority for that issue..

-- Profile update not working
--- Quantity field not working.
----Webview not loading.

Screenshots issues (required):
https://screencast-o-matic.com/watch/cYeYoLyhxK


Config files (required)
https://we.tl/t-q6wqvH7ES3
